app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
I agree with you, if UUID is used, authorisation seems to be unnecessary. I'll update then